src/consts.py

```diff
@@ -4,5 +4,18 @@
 """This module defines constants used throughout the Flask application."""
 
 FLASK_CONTAINER_NAME = "flask-app"
```

Comment:
`FLASK_CONTAINER_NAME` and `FLASK_SERVICE_NAME` could be defined in `charm.py` instead

Reply:
It can't be defined in `charm.py` since that will cause circular import.